### PR TITLE
Adding Show All menu option

### LINKED ISSUE #2058: No Settings > Settings > Microsoft 365 Groups in M 365 Admin Center
[I'm not able to find Settings > Settings > Microsoft 365 Groups in the Microsoft 365 Admin Center. This navigation does not exist in either the new admin center or the old.]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9f5592e2-3fee-9034-5b33-2d1687d60ba2
* Version Independent ID: 5b8dee58-e383-9a26-570a-47f5c81121cb
* Content: [Manage guest access in Microsoft 365 groups - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/create-groups/manage-guest-access-in-groups?view=o365-worldwide)
* Content Source: [microsoft-365/admin/create-groups/manage-guest-access-in-groups.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/create-groups/manage-guest-access-in-groups.md)
* Service: **o365-administration**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2058

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/create-groups/manage-guest-access-in-groups.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Guest access in groups is often used as part of a broader scenario that includes
 
 If you want to enable or disable guest access in groups, you can do so in the Microsoft 365 admin center.
 
-1. In the admin center, go to the **Settings** \> **Org settings** and on the **Services** tab, select **Microsoft 365 groups**.
+1. In the admin center, go to the **Show all** \> **Settings** \> **Org settings** and on the **Services** tab, select **Microsoft 365 groups**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/manage-guest-access-in-groups.md]:
```suggestion
1. In the admin center, go to **Show all** \> **Settings** \> **Org settings** and on the **Services** tab, select **Microsoft 365 groups**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@get-itips - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #2058.

Thank you.

**Pull request merged**